### ISSUE TITLE
Cross platform detection of the filesystem block size

### ISSUE THREAD

**Issue by iliafrenkel** [iliafrenkel/go-du#5]
At the moment the block size is hardcoded to be 4096 bytes as the most common.
There is a way to do it in Linux by calling [`syscall.Statfs`](https://golang.org/pkg/syscall/#Statfs):
```go
// Get filesystem block size.
//
// Returns block size in bytes or -1 in case of an error.
// https://man7.org/linux/man-pages/man2/statfs.2.html
func getFSBlockSize(path string) int64 {
	var stat syscall.Statfs_t
	if err := syscall.Statfs(path, &stat); err != nil {
		return -1
	} else {
		return stat.Bsize
	}
}
```
We need to either find a similar solution that is cross-platform or say that Windows is not supported. I am inclined to do the latter. After all, Windows is not POSIX compatible and I'm not aware of any `du` implementations that work on Windows.

**Comment by iliafrenkel** [comment]
Decision made - we're dropping Windows support.

**Comment by iliafrenkel** [comment]
Closed by #7
